README.rst

```diff
@@ -31,37 +31,36 @@ Note that you must replace "coolem" by your plugin name.
 
     rm CHANGES.txt && touch CHANGES.txt
 
-**Rename "myplugin" to coolem (IDE might help here)**
+**Rename "myplugin" to coolem and scipion-em-template to scipion-em-coolem**
```

Comment:
This shoudn't be necessary if you followed the clone step: git clone git@github.com:scipion-em/scipion-em-template.git scipion-em-coolem

Reply:
I think this part can go away too!

"and scipion-em-template to scipion-em-coolem"